action/actctx.go

```diff
@@ -54,6 +59,22 @@ func (act *AbstractAction) SetGasPrice(val *big.Int) {
 	act.gasPrice = val
 }
 
+// GasTipCap returns the gas tip cap
+func (act *AbstractAction) GasTipCap() *big.Int {
+	if act.gasTipCap == nil {
+		return act.GasPrice()
+	}
+	return new(big.Int).Set(act.gasTipCap)
+}
+
+// GasFeeCap returns the gas fee cap
+func (act *AbstractAction) GasFeeCap() *big.Int {
+	if act.gasFeeCap == nil {
+		return act.GasPrice()
+	}
+	return new(big.Int).Set(act.gasFeeCap)
+}
+
 // BasicActionSize returns the basic size of action
 func (act *AbstractAction) BasicActionSize() uint32 {
```

Comment:
the two new fields should be accounting in BasicActionSize()?

Reply:
fixed